README.md

```diff
@@ -38,7 +38,7 @@ Select 1:
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
+```JavaScript functions can only recognize variables declared within themselves or their parent function. This means variables declared within functions will not be recognized globally, and global variables will not be recognized by functions within functions unless they are scoped in using bind or self=this.
```

Comment:
:+1:
Global variable are available everywhere.  We use self/bind, to ensure we can reference an item that is not globally scoped.  Also, chaining of scopes applies to any outer function, not just the parent function.  Chaining of scopes. Any function defined within another function has a local scope which is linked to the outer function.  see: http://www.coolcoder.in/2014/03/everything-you-need-to-know-about.html